tests/job_tests/recognition/test_scoring.py

```diff
@@ -9,7 +9,7 @@
 from i6_core.recognition import ScliteJob
 
 Path = setup_path(__package__)
-
+_my_dir = os.path.dirname(os.path.abspath(__file__))

```

Comment:
Missing newline here.
(Shouldn't black fix this?)

Reply:
Whoops, didnt mean to commit all of this. But still, I guess both versions work, will fix black.